program = { env = "AUREUM_TEST_EXEC" }


[tests.show-all-tests]
program_arguments = [
  "--show-all-tests",
  "basic-helpers/passing-test.toml",
  "basic-helpers/failing-test.toml",
]
expected_stdout = """
🚀 Running 2 tests:
F.

❌ basic-helpers/failing-test.toml
├─ Standard output
│  ├─ Expected
│  │  ╭
│  │  │ STDOUT
│  │  ╰ (No newline at end)
│  └─ Got
│     ╭
│     │ Hello world
│     ╰ (No newline at end)
├─ Standard error
│  ├─ Expected
│  │  ╭
│  │  │ STDERR
│  │  ╰ (No newline at end)
│  └─ Got
│     ╭
│     │ 
│     ╰ (No newline at end)
└─ Exit code
   ├─ Expected
   │  └─ 42
   └─ Got
      └─ 0
✅ basic-helpers/passing-test.toml

Test result: FAIL (1 passed, 1 failed)
"""
expected_stderr = ""
expected_exit_code = 1


[tests.default]
program_arguments = ["basic-helpers/passing-test.toml", "basic-helpers/failing-test.toml"]
expected_stdout = """
🚀 Running 2 tests:
F.

❌ basic-helpers/failing-test.toml
├─ Standard output
│  ├─ Expected
│  │  ╭
│  │  │ STDOUT
│  │  ╰ (No newline at end)
│  └─ Got
│     ╭
│     │ Hello world
│     ╰ (No newline at end)
├─ Standard error
│  ├─ Expected
│  │  ╭
│  │  │ STDERR
│  │  ╰ (No newline at end)
│  └─ Got
│     ╭
│     │ 
│     ╰ (No newline at end)
└─ Exit code
   ├─ Expected
   │  └─ 42
   └─ Got
      └─ 0

Test result: FAIL (1 passed, 1 failed)
"""
expected_stderr = ""
expected_exit_code = 1
